assets/site.scss

```diff
@@ -70,45 +72,48 @@ $content-width: 65rem;
   --gutter-width: calc((100vw - 65rem) / 2);
 
   --link-color: #0066cc;
+
   // FIXME
```

Comment:
while we are at it, what is this FIXME about?

Reply:
I don't see that FIXME... was this from reviewing an intermediate commit?

Reply:
Yes.

Reply:
Apologies! I need to squash some of this down still.